### DISCUSSION TITLE
Do You have any question? Ask me.

### DISCUSSION THREAD

**Discussion by Diptenusarkar** [Diptenusarkar/Website-Template#326] [Q&A]
If you need any help ask me.

**Comment by Sypher0Dronzer** [comment]
Sir How do we get the label of hactober-accepted in our pull requests.
I did manage to get my pull request merged, the one with ecommerce website one but it didnt have that tag.

**Reply by Diptenusarkar** [reply]
We don't have any criteria for that. If You PR Get approve you can Request for Label. We will comply with your request as soon as possible.
If You have any Questions ask Me. I'm here to support.
